pydantic/main.py

```diff
@@ -506,60 +514,6 @@ def validate(cls: Type['Model'], value: Any) -> 'Model':
     def _decompose_class(cls: Type['Model'], obj: Any) -> GetterDict:
         return cls.__config__.getter_dict(obj)
 
-    @classmethod
-    @no_type_check
-    def _get_value(
```

Comment:
Would you consider changing this from a method to a function to be a breaking change? (Given it is part of the private API?) If so, is it too late to change it for v1?

I can change things back to this being a method if necessary.